packages/grid/devspace.yaml

```diff
@@ -68,28 +65,34 @@ deployments:
       values:
         syft:
           registry: ${CONTAINER_REGISTRY}
-          version: "dev-latest"
+          version: dev-${DEVSPACE_TIMESTAMP}
         node:
           settings:
             nodeName: ${NODE_NAME}
             nodeType: "domain"
-        # configuration:
-        #   devmode: True
+        configuration:
```

Comment:
We need the dev mode to be enabled during only dev.k8s.hotreload, and  disable during dev.k8s.deploy

Reply:
see the comment on this issue https://github.com/OpenMined/Heartbeat/issues/913